src/pages/Locations/LocationDetail/LocationDetailContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import type { FC } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from 'src/components';
+import useLocationDetail from './hooks/useLocationDetail';
+import LocationDetail from './LocationDetail';
+
+type LocationQueryParam = {
+  locationId: string;
+};
+
+const LocationDetailContainer: FC = () => {
+  const { locationId } = useParams<LocationQueryParam>();
+  const result = useLocationDetail(locationId);
+
+  return <>{result.loading ? <Loader /> : <LocationDetail {...result} />}</>;
```

Comment:
no conditions/logic should be inside `*Container` components
Container is only for bounding hook (logic) and component (presentation)

Reply:
the same logic you have inside src/pages/Locations/LocationDetail/LocationDetail:33